internal/topo/planner/planner_graph.go

```diff
@@ -498,6 +498,7 @@ func parseSource(nodeName string, gn *api.GraphNode, rule *api.Rule, store kv.Ke
 			return nil, ILLEGAL, "", err
 		}
 		sourceOption.TYPE = gn.NodeType
+		sourceOption.SCHEMAID = gn.Props["schemaName"].(string) + "." + gn.Props["schemaMessage"].(string)
```

Comment:
Should only override schemaId if both schemaName and schemaMessage are set. Additionally, need to check if the props are string type or they will be a panic here.